pythonosc/osc_packet.py

```diff
@@ -68,7 +68,14 @@ def __init__(self, dgram: bytes) -> None:
                     key=lambda x: x.time,
                 )
             elif osc_message.OscMessage.dgram_is_message(dgram):
-                self._messages = [TimedMessage(now, osc_message.OscMessage(dgram))]
+                self._messages = []
+                while(True):
+                    msg = osc_message.OscMessage(dgram)
+                    self._messages.append(TimedMessage(now, msg))
+                    if msg._dgram_tail is None:
+                        break
+                    dgram = msg._dgram_tail
```

Comment:
please provide a readonly property instead of accessing the _dram_tail from another class